### DISCUSSION TITLE
So helpful having local control!

### DISCUSSION THREAD

**Discussion by kevinrossen** [cryptk/haomnilogic-local#120] [General]
Just wanted to say a quick thank you for developing this. This was one of the first automations I setup when I got started with Home Assistant and it's been wonderful being able to control our pool/hot tub from a Zooz Scene Controller. It's also been super handy to have while Hayward has been recovering from the Crowdstrike outage.

**Comment by mnka00** [comment]
Would you or someone be able to help me set this up? This is exactly what I am looking for but I have no idea where to start with HA and this code.

**Reply by kevinrossen** [reply]
Just now saw your comment, sorry! Yes, I'm happy to help.
[This is the scene controller I bought from The Smartest House.](https://www.thesmartesthouse.com/collections/zooz/products/zooz-700-series-z-wave-plus-scene-controller-switch-zen32?variant=40447790710847)
I can post a screenshot of my settings, too. I'll do that after I get the integration updated and running again.

**Comment by cryptk** [comment]
Sorry for the *SUPER* late response, but thanks! I just released 1.0.0 of the integration which is a complete overhaul under the hood (but shouldn't look *too* different to the end users, but there are a few breaking changes).

I also have my pool tied into a Zooz scene controller through this integration and it's been working fantastic! I haven't used the Hayward app in quite some time aside from troubleshooting/debugging while working on adding functionality to the integration.
